app/src/main/java/com/neongarage/Fragments/CountInFragment.java

```diff
@@ -136,29 +142,62 @@ public void handleSubmitClick(View v) {
         double dimeTotal = dimes.getTotal();
         double quarterTotal = quarters.getTotal();
         double coinValue = nickleTotal + quarterTotal + dimeTotal;
-        coinTotal.setText("Nickels: " + df.format(nickleTotal) + "\nDimes: " + df.format(dimeTotal) + "\nQuarters: " + df.format(quarterTotal));
+        coins.append("Nickels: ");
+        coins.append(df.format(nickleTotal));
+        coins.append("\nDimes: ");
+        coins.append(df.format(dimeTotal));
+        coins.append("\nQuarters: ");
+        coins.append(df.format(quarterTotal));
+        coinTotal.setText(coins);
+        coins.setLength(0);
 
         //Count value of dollars
         double oneTotal = ones.getTotal();
         double fiveTotal = fives.getTotal();
         double tenTotal = tens.getTotal();
         double twentyTotal = twenties.getTotal();
         double dollarTot = oneTotal + fiveTotal + tenTotal + twentyTotal;
-        dollarTotal.setText("Ones: " + df.format(oneTotal) + "\nFives: " + df.format(fiveTotal) + "\nTens: " + df.format(tenTotal) + "\nTwenties: " + df.format(twentyTotal));
+        dollar.append("Ones: ");
+        dollar.append(df.format(oneTotal));
+        dollar.append("\nFives: ");
+        dollar.append(df.format(fiveTotal));
+        dollar.append("\nTens: ");
+        dollar.append(df.format(tenTotal));
+        dollar.append("\nTwenties: ");
+        dollar.append(df.format(twentyTotal));
+        dollarTotal.setText(dollar);
+        dollar.setLength(0);
 
         //Grand total
         double grandTot = dollarTot + coinValue;
-        grandTotal.setText("Grand Total: " + df.format(grandTot));
+        grand.append("Grand Total: ");
+        grand.append(df.format(grandTot));
+        grandTotal.setText(grand);
+        grand.setLength(0);
 
         //Difference
         if (grandTot > 150.45) {
             double difference = grandTot - 150.45;
-            resultText.setText("You are over\nDrop: " + df.format(Math.ceil(difference)));
+            //Shay says this is green....
+            resultText.setTextColor(Color.rgb(0, 128, 0));
+            result.append("You are over ");
+            result.append(df.format(Math.ceil(difference)));
+            resultText.setText(result);
+            result.setLength(0);
         } else if (grandTot < 149.95) {
             double difference = 149.95 - grandTot;
-            resultText.setText("You are under. \nAsk Gatekeeper for " + df.format(Math.ceil(difference)));
-        } else
-            resultText.setText("You are right on the Money. Good Job");
+            //Apparently this is red
+            resultText.setTextColor(Color.rgb(128, 0, 0));
```

Comment:
You should use the defined colors attributes or add them there       resultText.setTextColor(R.attr.colorPrimaryDark);

Reply:
You can use 
        <item name="colorPrimaryDark">@color/colorPrimaryDark</item> -- for  green
        <item name="colorAccent">@color/colorAccent</item> -- for red